### ISSUE TITLE
Add option to reverse shortcuts for zoom and image switching

### ISSUE THREAD

**Issue by mdlt7z** [netdcy/FlowVision#36]
Hi,
I have been using your app since it was published :) 

I usually use Illustrator, Figma, etc. and am used to scaling only with the mouse wheel.

As a suggestion, I thought it would be nice to have an option to swap the shortcut for the current zoom and the shortcut for switching images.

- normal scroll to switch images
- button press + scroll to zoom

switching option ↕️

- normal scroll to zoom
- button press + scroll to switch images

Thanks for a great app!

**Comment by netdcy** [comment]
Hi, thank you for your suggesstion. 

Currently, the program switches by holding down the right or left mouse button. Do you mean, for example, making the Cmd key have the same effect as holding the right mouse button? And also adding an option in settings to reverse this by default?

**Comment by mdlt7z** [comment]
It is not strictly necessary to implement it as described above, but what I ultimately want to achieve is the ability to zoom in and out of the image using only the mouse wheel, without having to press any buttons.

Currently, scrolling the mouse wheel only switches between images, but I feel it would be useful to have the option to assign this action to zooming. However, if this is done, image switching would no longer be possible, so I think it might be better to reassign the original zoom gesture (button hold + wheel) to something else.

I also like the idea of utilizing the Cmd key! It might make the operation easier for some users. However, for me personally, the preference remains as noted above.

**Comment by netdcy** [comment]
Yes, I understand that everyone has different preferences, and many people have requested customizable keys.

But I haven't found a convenient package to implement this feature, and it requires a certain amount of work, especially related to mouse gestures (this is my first time developing a macOS application!). It might be added in the next major version.

**Comment by netdcy** [comment]
This has been added in ver 1.6.6.
